CONTRIBUTING.md

```diff
@@ -2,38 +2,30 @@
 
 ## Setup
 
-In order to contribute to librespot, you will first need to set up a suitable rust build environment, with the necessary dependenices installed. These instructions will walk you through setting up a simple build environment.
+In order to contribute to librespot, you will first need to set up a suitable Rust build environment, with the necessary dependencies installed. These instructions will walk you through setting up a simple build environment.
 
-You will need to have C compiler, rust, and portaudio installed.
+You will need to have C compiler, Rust, and alsa/portaudio libraries installed.
```

Comment:
s/have C compiler/have a C compiler/


It's also better to say something like "and the development libraries for the audio backend(s) you want". You don't need any ALSA/PortAudio libraries if you are going with the default Rodio backend.

Reply:
> Rodio uses Cpal which in turn uses Alsa/Winapi, so will still need ALSA right?

I've never tried to build the Rodio backend but I assume not. Sounds like that assumption was total rubbish. In which case:

s/alsa/ALSA/
s/PortAudio/